### ISSUE TITLE
Update to Angular 2 RC

### ISSUE THREAD

**Issue by KarlSjostrand** [joost-de-vries/play-angular2-typescript#8]
There are now Angular 2 RC.3 deps at http://www.webjars.org/npm. Would it be possible to update this template accordingly?

**Comment by Alcalion** [comment]
Hi there !

I started a project a few months ago and used this project to build my angular2 properly.
I can still work with the angular2 beta version but upgrading it to the RC would really help. Hence I started the update but unfortunately I keep facing the same import issue in my ts files.

I think there is a major difference in the fact that you now have to use several webjars instead of just one previously. I will keep an eye on this issue if there are any updates, and if I find a solution until then I'll come share it here.
